docs/tutorial/from-zero-to-hero-write-your-first-kubernetes-charm/integrate-your-charm-with-postgresql.md

```diff
@@ -65,20 +63,18 @@ requires:
 
 That will tell `juju` that our charm can be integrated with charms that provide the same `postgresql_client` interface, for example, the official PostgreSQL charm.
 
-
 Import the database interface libraries and define database event handlers
 
-We now need to implement the logic that wires our application to a database. When a relation between our application and the data platform is formed, the provider side (i.e., the data platform) will create a database for us and it will provide us with all the information we need to connect to it over the relation -- e.g., username, password, host, port, etc. On our side, we nevertheless still need to set the relevant environment variables to point to the database and restart the service.
+We now need to implement the logic that wires our application to a database. When a relation between our application and the data platform is formed, the provider side (that is: the data platform) will create a database for us and it will provide us with all the information we need to connect to it over the relation -- for example, username, password, host, port, and so on. On our side, we nevertheless still need to set the relevant environment variables to point to the database and restart the service.
```

Comment:
I feel that "data platform" is jargon that only Warthogs understand.

Arguably the whole paragraph needs to be split into smaller, and the sentences into shorter but more numerous too.

Maybe it's worth breaking this into steps, something like:

- administrator integrates your app with a database app
- the database charm notices the new relation
- it creates a new logical database for your app
- it publishes the address and credentials on the relation
- your app picks these up from the relation
- your app can now talk to the database

Reply:
These all sound good to me ... but I think I already caused issues by putting changes in this PR that aren't "removing some chapters". What about we do the above in a follow-up PR instead?